src/block/tab-labels/style.js

```diff
@@ -125,7 +129,7 @@ const Styles = props => {
 				} }
 				attrName="contentAlign"
 				key="iconAlignment-iconPosition"
-				enabledCallback={ getAttribute => getAttribute( 'fullWidth' ) || getAttribute( 'iconPosition' ) === 'top' || getAttribute( 'iconPosition' ) === 'bottom' }
+				enabledCallback={ getAttribute => getAttribute( 'fullWidth' ) !== undefined || getAttribute( 'iconPosition' ) === 'top' || getAttribute( 'iconPosition' ) === 'bottom' }
```

Comment:
Is this correct? I remember seeing initially this in your earlier commit:

```js
getAttribute( 'fullWidth' ) || ! getAttribute( 'fullWidth' ) || getAttribute( 'iconPosition' ) === 'top' || getAttribute( 'iconPosition' ) === 'bottom' ```

which can be simplified to

```js
getAttribute( 'iconPosition' ) === 'top' || getAttribute( 'iconPosition' ) === 'bottom'
```

..But this new commit has `getAttribute( 'fullWidth' ) !== undefined`